other/edge-server/README.md

```diff
@@ -306,13 +329,13 @@ connect ECONNREFUSED ::1:27021, connect ECONNREFUSED 127.0.0.1:27021
 ```
 
 This indicates that the Edge Server is not running on port 27021. Verify
-that the Edge Server is running. If you have changed the URI and/or port where 
-the Edge Server is listening for wireprotocol connections, change the URI 
+that the Edge Server is running. If you have changed the URI and/or port where
+the Edge Server is listening for wireprotocol connections, change the URI
 and/or port in your `.env` file.
 
 ## Install dependencies and start the React server
 
-With the Edge Server and the Express server running, cd into the `react-client` 
+With the Edge Server and the Express server running, cd into the `react-client`
```

Comment:
We can probably just delete this section about installing dependencies and starting the React server, since that can now be managed at the higher level with your new package.json.

(Also, we should document that as the way to install the dependencies and run the servers.)

Reply:
I reframed this section and the Express section as alternatives to the quick start and I link to the quick start above. The info in these sections still works if people want to do it on their own.

Reply:
Just kidding. I've merged the Express and React sections and made it more of the happy path.